{"name": "test_get_non_existing_contact", "status": "passed", "description": "Test getting a non-existing contact.", "attachments": [{"name": "log", "source": "9d068a9e-61db-48ac-830c-0c6872b9e7b3-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "8e4e1bbf-72a8-4db9-8c23-f1eb22f50098-attachment.txt", "type": "text/plain"}], "start": 1737939449674, "stop": 1737939450199, "uuid": "c142fbb6-e428-4289-9431-457b90ed44dc", "historyId": "dc4b7ce8a52a689c10fe51185ce4d787", "testCaseId": "dc4b7ce8a52a689c10fe51185ce4d787", "fullName": "tests.api_tests.contacts.test_contacts_api_negative#test_get_non_existing_contact", "labels": [{"name": "parentSuite", "value": "tests.api_tests.contacts"}, {"name": "suite", "value": "test_contacts_api_negative"}, {"name": "host", "value": "Dima"}, {"name": "thread", "value": "20172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.contacts.test_contacts_api_negative"}]}